src/react/components/GenericForm.jsx

```diff
@@ -86,39 +86,65 @@ export class GenericForm extends React.Component {
     }
 
     /**
-     * @method _createFormItems
+     * @method _createFormComponents
      * @private
      * 
-     * Crate single form items
+     * Create all form components
+     * 
+     * @param {Object[]} fields - array containing all fields
      * 
-     * @param {Object} fields - array containing all fields
      * @returns {React.Component} 
      */
-    _createFormItems (fields, formSettings) {
+    _createFormComponents (fields) {
         return fields.map((field) => {
-            return (
-                <StyledFormItem
-                    key={ field.name }
-                    name={ field.type !== 'buttongroup' ? field.name : '' }
-                    label={ formSettings.showLabels ? field.label : '' }
-                    rules={ field.rules }
-                    initialValue={ this._getFieldInitialValue(field) }
-                >
-                    { this._createFormInput( field ) }
-                </StyledFormItem>
-            );
+            if(field.showTooltip) {
+                return (
+                    <Tooltip title={ field.placeholder || field.label } trigger={ ['focus', 'hover'] }>
+                        { this._createFormItem( field ) }
+                    </Tooltip>
+                );
+            } else {
+                return this._createFormItem( field );
+            }
```

Comment:
Minor thing but doing it this way makes it clear the form item itself is always created the same way:

```suggestion
    const formItem = this._createFormItem(field);
    if (field.showTooltip) {
        return addTooltip(formItem, field.placeholder || field.label);
    }
    return formItem;
```

This could be added somewhere earlier in the file (doesn't need to be inside the react component):
```
const addTooltip = (formItem, text) => {
    if (!text) {
        return formItem;
    }
    return (<Tooltip title={ text } trigger={ ['focus', 'hover'] }>{ formItem }</Tooltip>);
}
```

Reply:
I would also change showTooltip from boolean to "tooltip" with a value of the tooltip text that we want to show. Then the caller can decide what to show as tooltip instead of it being placeholder or label.

Reply:
Check!